modules/creating-instance-aws-load-balancer-controller.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/create-instance-aws-load-balancer-controller.adoc
+
+:_content-type: PROCEDURE
+[id="nw-creating-instance-aws-load-balancer-controller_{context}"]
+= Creating an instance of the AWS Load Balancer Controller using ALB Operator
+
+After installing the Operator, you can create an instance of the `AWSLoadBalancerController`. You can install only a single instance of the `aws-load-balancer-controller` in a cluster. The Operator reconciles only the resource with the name `cluster`.
+
+.Prerequisites
+
+. You have created the `echoserver` namespace.
+
+.Procedure
+
+. Create a `AWSLoadBalancerController` resource:
++
+[source,yaml]
+----
+apiVersion: networking.olm.openshift.io/v1alpha1
+kind: AWSLoadBalancerController <1>
+metadata:
+  name: cluster <2>
+spec:
+  subnetTagging: Auto <3>
+  additionalResourceTags: <4>
+    example.org/cost-center: 5113232
+    example.org/security-scope: staging
+  ingressClass: cloud <5>
+  config:
+    replicas: 2 <6>
+  enabledAddons: <7>
+    - AWSWAFv2 <8>
+----
+
+<1> Defines the `AWSLoadBalancerControlller` resource.
+<2> Defines the `aws-load-balancer-controller` instance name. This instance name gets added as a suffix to all related resources.
+<3> Valid options are `Auto` and `Manual`. When the value is set to `Auto` the Operator attempts to determine the subnets that belong to the cluster and tags them appropriately. The Operator cannot determine the role correctly if the internal subnet tags are not present on internal subnet. If you installed your cluster on user-provided Infrastructure, you must manually tag the subnets with the appropriate role tags and set the subnet tagging policy to `Manual`.
```

Comment:
It is not always true for UPI, because UPI can also set proper tags for subnet during/after installation.
see https://github.com/openshift/aws-load-balancer-operator/pull/58/files

And we also should mention the tag for VPC on UPI some where.

Reply:
@lihongan I added the sentence as `Additionally,  user-provisioned infrastructure installation can also set tags for subnet during or after installation. ` Let me know if it needs a change. 

And can you please elaborate about adding a tag for VPC on UPI ? Sorry I didn't understand what to add exactly.

cc: @arjunrn

Reply:
The installation instructions mention that the tag should be present on the VPC. It is mentioned in the installation instructions over [here](https://github.com/openshift/openshift-docs/blame/c1f919bb0dfdd16ada9df9147a7a008850bab96d/modules/installing-aws-load-balancer-operator.adoc#L21). Since this text is just for the subnets we added a warning here again. But I don't think the VPC tagging should be mentioned here again.

Also the instructions here are correct, after a UPI install we don't want the operator to perform the tagging again, so it should be set to Manual.

Reply:
Ack!